cmd/kogito/command/install/jobs_service.go

```diff
@@ -37,6 +38,7 @@ const (
 type installJobsServiceFlags struct {
 	deploy.CommonFlags
 	image                         string
+	insecureImageRegistry         bool
```

Comment:
Maybe having `image` and `insecureImageRegistry` on their own structure? This way we can reuse some code. Once we have implemented the Runtime and Build in the CLI we will be able to reuse it as well for the `deploy` command.

Reply:
Hmm, does it have sense to put `image` and `insecureImageRegistry` into one structure here? In `kogitoservices_types.go` we keep them separate.

Reply:
They do not follow the same context, IMO. Those are flags reusable across many commands, adding them in the same structure would improve reusablity.

Reply:
ok, will try...